go.mod

```diff
@@ -131,3 +131,5 @@ require (
 	sigs.k8s.io/kube-storage-version-migrator v0.0.4 // indirect
 	sigs.k8s.io/structured-merge-diff/v4 v4.2.0 // indirect
 )
+
+replace github.com/openshift/api => github.com/yannickstruyf3/api v0.0.0-20211116212905-775f773bafe0
```

Comment:
This will be removed once [API PR](https://github.com/openshift/api/pull/1059) is merged